CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
```

Comment:
Sure no problem. I can add it in a separate commit. It does make history a bit more readable.